docs/application/dotnet/guides/alarm/alarms.md

```diff
@@ -56,6 +56,9 @@ To enable your application to use the alarm functionality:
 
 You can set an alarm which, when it expires, either launches an application or sends a notification to the user:
 
+> [!NOTE]
+> Since Tizen 6.0, the time period value of an alarm can be one of the values of `Tizen.Applications.AlarmStandardPeriod`. For the `CreateAlarm()` method of the `Tizen.Applications.AlarmManager` class, if you use `Tizen.Applications.AlarmStandardPeriod`, the time period value of the alarm is guaranteed. If not, the time period value will be phase-aligned with another time period value of the alarm.
```

Comment:
Doubt:
As you have changed the sentence flow, before approving, I would like to clear my doubts.

In the last part of the sentence, are we trying to say
...the time period value of the alarm is guaranteed. If not guaranteed, then the time period value will be phase-aligned with another time period value of the alarm.

Please verify and let me know if this is correct.

Reply:
If not `Tizen.Applications.AlarmStandardPeriod, then the time period value will be phase-aligned with another time period value of the alarm.

I think it is better

Reply:
Actually I think "If not" means "If `Tizen.Applications.AlarmStandardPeriod` is not used".
as using `Tizen.Applications.AlarmStandardPeriod` guarantee the time period value.

The previous sentence is "If you use `Tizen.Applications.AlarmStandardPeriod`...",
I think starting the next sentence with "If not, " is not bad.

Reply:
@kilig 
Thank you for clearing my doubt.

@mijin85cho 
Using "if not" is not bad and the sentence is fine.
But as the sentence is long and is separated, so can we make it more clear as:

For the `CreateAlarm()` method of the `Tizen.Applications.AlarmManager` class, if you use `Tizen.Applications.AlarmStandardPeriod`, the time period value of the alarm is guaranteed. If `Tizen.Applications.AlarmStandardPeriod` is not used, the time period value will be phase-aligned with another time period value of the alarm.

@kilig  and @mijin85cho,
Please verify.

Reply:
I've updated

Reply:
Thank you for updating the changes.

I have responded to your suggestion for @mijin85cho's comment.
For more information, see https://github.com/Samsung/tizen-docs/pull/1078#issuecomment-617572136.
Please verify the same.